integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/ReplaceKeys.feature

```diff
@@ -19,6 +19,41 @@ Feature: SmartMetering Configuration - Replace Keys
     And the new keys are stored in the secret management database encrypted_secret table
     And the stored keys are not equal to the received keys
 
+  Scenario: Replace keys on a device that has previously created keys with status NEW
```

Comment:
Suggestion: change scenario description to make clear that the other NEW keys are older than the specified max time in the properties.

Reply:
Changed the scenarion description to 
_Replace keys on a device that has previously created keys with status NEW **older than the specified max time in the properties**._